source/_posts/2022-12-20-year-of-voice.markdown

```diff
@@ -0,0 +1,65 @@
+
+<center><img src='/images/blog/2022-12-20-year-of-voice/voice-work.png' alt='Spoken languages times possible actions' class='no-shadow' /></center>
+<br>
+
+Our #1 priority is supporting different languages. There are enough projects out there trying to create an English voice assistant. But for us that just doesn't cut it. People need to be able to speak in their own language, as that is the most accessible and only acceptable language for a voice assistant for the smart home.
+<!--more-->
+
+To keep the amount of work ahead of us manageable, we're going to limit the number of possible actions and focus on the basics of interacting with your smart home. No web searches, making calls or voice games. And definitely no "by the way"s!
+
+We are going to start with a few actions and build up the language models around that. Home Assistant supports 62 different languages in its user interface. And it's our goal to support all these languages with voice. We think that we can achieve that by leveraging Home Assistant's strongest asset: our community.
+
+## Our history with voice assistants
+
+If you follow the news, it might sound like voice assistants have failed. Amazon is set to [lose $10 billion on Alexa this year](https://arstechnica.com/gadgets/2022/11/amazon-alexa-is-a-colossal-failure-on-pace-to-lose-10-billion-this-year/) and is planning layoffs. Google too, is reducing its [support for Google Assistant](https://arstechnica.com/gadgets/2022/10/report-google-doubles-down-on-pixel-hardware-cuts-google-assistant-support/) as it’s trying to cut costs. The truth is that voice as the next computing platform that drives billions of dollars of extra revenue has failed. Instead, users mainly use their voice assistants to manage shopping lists, set timers, play music and control their home. Voice has failed being a source of revenue, it has not failed its users.
+
+With Home Assistant we’ve always been interested in voice. We used to work with [Snips](https://snips.ai/) back in the day, but they got acquired and shut down. We worked with Stanford on their [Almond/Genie platform](https://www.home-assistant.io/blog/2021/12/21/stanford-genie/), but it is a research driven project that never got production ready. And yes, you can use Home Assistant to send all your data to the clouds of Google and Amazon to leverage their voice assistants, but you shouldn't have to give up your privacy to just turn on the lights by voice.
+
+The most promising project out there is [Rhasspy](https://rhasspy.readthedocs.io/en/latest/) created by Mike Hansen. A project that allowed people to build their own voice assistant which could also tie into Home Assistant. Rhasspy stands out from other open source voice projects because Mike didn’t focus on just English. Instead, his goal has been to make it work for everyone and he already supports 16 different languages today.
```

Comment:
```suggestion
The most promising project out there is [Rhasspy](https://rhasspy.readthedocs.io/en/latest/) created by Mike Hansen. A project that allowed people to build their own voice assistant, which could also tie into Home Assistant. Rhasspy stands out from other open source voice projects because Mike didn’t focus on just English. Instead, his goal has been to make it work for everyone and he already supports 16 different languages today.
```